### PR TITLE
Document the ProductOpener::Mail.pm module

### PR DESCRIPTION
**Pull request by roshnaeem**
Description: Updated the `Mail.pm` file to add its documentation in perlpod format.

Fixes: https://github.com/openfoodfacts/openfoodfacts-server/issues/5084

Related Issue: https://github.com/openfoodfacts/openfoodfacts-server/issues/2203

### CODE REVIEW

File: lib/ProductOpener/Mail.pm [hunk 1]
```diff
@@ -68,6 +115,25 @@ sub send_email($$$)
...
 
+=head1 FUNCTIONS
+
+=head2 send_email_to_admin( SUBJECT, TEXT )
+
+C<send_email_to_admin()> sends email from the contact email of Open Food Facts to the admin.
+
+=head3 Arguments
+
+Two scalar varaibles that contain the email subject and body are passed as an argument.
```
Comment (CharlesNepote) [inline lib/ProductOpener/Mail.pm:126]:
```suggestion
Two scalar variables that contain the email subject and body are passed as an argument.
```

File: lib/ProductOpener/Mail.pm [hunk 2]
```diff
@@ -85,6 +151,25 @@ sub send_email_to_admin($$)
...
 
+=head1 FUNCTIONS
+
+=head2 send_email_to_producers_admin( SUBJECT, TEXT )
+
+C<send_email_to_producers_admin()> sends email from the contacts email of Open Food Facts to producers admin.
+
+=head3 Arguments
+
+Two scalar varaibles that contain the email subject and body are passed as an argument.
```
Comment (CharlesNepote) [inline lib/ProductOpener/Mail.pm:162]:
```suggestion
Two scalar variables that contain the email subject and body are passed as an argument.
```

File: lib/ProductOpener/Mail.pm [hunk 3]
```diff
@@ -85,6 +151,25 @@ sub send_email_to_admin($$)
     return $@ ? 1 : 0;
 }
 
+=head1 FUNCTIONS
+
+=head2 send_email_to_producers_admin( SUBJECT, TEXT )
+
+C<send_email_to_producers_admin()> sends email from the contacts email of Open Food Facts to producers admin.
```
Comment (CharlesNepote) [inline lib/ProductOpener/Mail.pm:158]:
```suggestion
C<send_email_to_producers_admin()> sends email from the contact email of Open Food Facts to producers admin.
```

File: lib/ProductOpener/Mail.pm [hunk 4]
```diff
@@ -44,6 +65,32 @@ use ProductOpener::Lang qw/:all/;
...
 
+=head1 FUNCTIONS
+
+=head2 send_email( USER_REF, SUBJECT, TEXT )
+
+C<send_email()> sends email from the contact email of Open Food Facts to the email passed as an argument.
+
+=head3 Arguments
+
+The first argument is a hash reference. The other two arguments are scalar varaibles that consist of email subject and body.
```
Comment (CharlesNepote) [inline lib/ProductOpener/Mail.pm:76]:
```suggestion
The first argument is a hash reference. The other two arguments are scalar variables that consist of email subject and body.
```

**Review by CharlesNepote — CHANGES_REQUESTED** [review]
Thanks! I fixed small typos.

**Comment by roshnaeem** [comment]
> Thanks! I fixed small typos.

Thank you so much for your feedback.

**Review by stephanegigandet — APPROVED** [review]
Looks good to me, thank you!

**Review by CharlesNepote — APPROVED** [review]
All right!

**Pull request merged**